lib/extensions.js

```diff
@@ -89,8 +89,9 @@ function getBinaryUrl() {
   return flags['--sass-binary-url'] ||
          package.nodeSassConfig ? package.nodeSassConfig.binaryUrl : null ||
```

Comment:
Agree. Then lets replace `sassBinaryUrl` with sassBinarySite (or preferably `sassBinaryCdn` as per the original requester https://github.com/sass/node-sass/issues/720) with the following precedence order:

* CLI argument `--sass-binary-cdn`
* `nodeSassConfig.sassBinaryCDN`
* `process.env.sassBinaryCDN`

If none of them are found, it will fall back to default github URL.